api/v1/runtime/runtime_agent_api.proto

```diff
@@ -342,3 +344,40 @@ message KubeLinterCheck {
   uint64 passed = 2; // Represented as bitmasks of passed checks.
   uint64 failed = 3; // Represented as bitmasks of failed checks.
 }
+
+enum NetflowProtocol {
+  NETFLOW_PROTOCOL_UNKNOWN = 0;
+  NETFLOW_PROTOCOL_TCP = 1;
+  NETFLOW_PROTOCOL_UDP = 2;
+}
+
+message Netflow {
+  uint64 start_ts = 1; // Stored as unix timestamp in nanoseconds.
+  uint64 end_ts = 2; // Stored as unix timestamp in nanoseconds.
+  string process_name = 3;
+  string namespace = 4;
```

Comment:
Would it make sense to extract namespace, pod_name, container_name, workload_name, workload_kind into its own message? It is used in quite a few places.

Reply:
Flow destination contains all fields except container_name. IMO It would like a bit strange to have mixed approach.

Reply:
I overlooked the missing `container_name`, so fine with me 👍